bridgetown-core/test/test_configure_command.rb

```diff
@@ -52,5 +54,49 @@ def configurations_path
 
       File.delete("#{configurations_path}/roar.rb")
     end
+
+    should "perform configuration when option provided" do
+      output = capture_stdout do
+        @cmd.invoke(:perform_configurations, ["test_configuration_with_option:github"])
+      end
+
+      frontend_styles_path = File.expand_path(root_dir("frontend", "styles"), __dir__)
+
+      assert_match %r!create.*syntax-test.css!, output
+      assert File.exist?("#{frontend_styles_path}/syntax-test.css")
+
+      File.delete("#{frontend_styles_path}/syntax-test.css")
+    end
+
+    should "perform multiple configurations when option provided" do
+      capture_stdout do
+        @cmd.invoke(:perform_configurations, ["test_configuration_with_option:github test_automation"])
+      end
+
+      frontend_styles_path = File.expand_path(root_dir("frontend", "styles"), __dir__)
+      assert File.exist?("#{frontend_styles_path}/syntax-test.css")
+      # TODO: This test doesn't work, but manual testing seems to prove it does work fine
+      # I _think_ it's something to do with capture_stdout not capturing everything... maybe?
+      # assert_match %r!fixture.*?Works\!!, output
+      # assert_match %r!create.*syntax-test.css!, output
+
+      File.delete("#{frontend_styles_path}/syntax-test.css")
+    end
+
+    should "show error when option doesn't exist" do
+      output = capture_stdout do
+        @cmd.invoke(:perform_configurations, ["test_configuration_with_option"])
+      end
+
+      assert_match %r!Theme not valid!, output
+    end
+
+    should "show error when option is invalid" do
+      output = capture_stdout do
+        @cmd.invoke(:perform_configurations, ["test_configuration_with_option:invalid"])
+      end
+
+      assert_match %r!Theme not valid!, output
+    end
```

Comment:
These tests were useful to me to ensure it was working well, but technically it's testing stuff within the test file as opposed to running the command, so not sure how valuable they are?

Reply:
Yeah I agree this is testing the fixture rather than the actual command so I don't see any value it in personally.

Reply:
For these tests I'd recommend using a very simple test configuration. Similar to `test_automation.rb`. All we want to test is that the options were passed to the configuration file successfully. What the configuration script does with the options is not relevant to this test. Does that make sense?

These tests should be testing the `configure` command in a generic way.